lib/mirage.mli

```diff
@@ -457,12 +457,18 @@ type conduit
 val conduit: conduit typ
 val conduit_direct: ?tls:bool -> stackv4 impl -> conduit impl
 
-(** {2 HTTP configuration} *)
+(** {2 HTTP (Cohttp) configuration} *)
 
 type http
 val http: http typ
 val http_server: conduit impl -> http impl
 
+(** {2 HTTP (http/af) configuration} *)
+
+type httpaf
+val httpaf: httpaf typ
+val httpaf_server: conduit impl -> httpaf impl
```

Comment:
I'm curious why/whether there's need for `type httpaf` -- if we'd reuse `type http`, `httpaf` could be a drop-in replacement for `cohttp` -- similarly to how `arp` (using https://github.com/mirage/mirage-tcpip/tree/master/src/arpv4) and `farp` (using https://github.com/hannesm/arp) are defined.

Reply:
There probably isn't. I mostly didn't know what I was doing until everything compiled and ran.
I'll clean this up, I agree that `httpaf_server` could probably be:

`val httpaf_server: conduit impl -> http impl`

Reply:
yes, I just added links to how arp is done in mirage.mli / mirage_impl_arpv4. thanks!